src/arm/midr.h

```diff
@@ -167,6 +168,11 @@ inline static bool midr_is_kryo_gold(uint32_t midr) {
 	return (midr & uarch_mask) == (CPUINFO_ARM_MIDR_KRYO_GOLD & uarch_mask);
 }
 
+inline static bool midr_is_ampere_altra(uint32_t midr) {
```

Comment:
I am not too familiar with Ampere altra line of CPUs, is the function name sufficient to uniquely identify this particular part i.e. do we need some `plus` or `gold` or some suffix :)

Reply:
Altra is based on Cortex-A76 (Pixel4 big core), so as far uarch goes, XNNPack shares kernels that work well on Pixel 4.

But cache will be different

Our 'A75' kernels are actually tuned for A76, since A75 doesnt care much how you load, but A76 does.

The Neoverses are similar
V1 = X1
N1 = A710
V2 = X3
N2 = A720

On Intel I'm doing the same with granite rapids uarch is the same as Meteor lake.  microkernel instruction behavior is sensitive to the uarch.

Knowing the shared uarch makes development easier.  If you want to optimize for Ampere Altra you can test on Pixel 4 or vice versa.

Reply:
The Altra Ampere seems to be its own thing, i.e. not a _line_ of CPUs: https://amperecomputing.com/briefs/ampere-altra-family-product-brief.